mcu-util/src/messaging/can/canfd.rs

```diff
@@ -201,26 +201,20 @@ impl MessagingInterface for CanRawMessaging {
         };
 
         if let Some(bytes) = bytes {
-            let mut buf: [u8; 64] = [0u8; 64];
+            let mut buf: [u8; CANFD_DATA_LEN] = [0u8; CANFD_DATA_LEN];
             buf[..bytes.len()].copy_from_slice(bytes.as_slice());
 
             let node_addr = self.can_node as u32;
             let frame = Frame {
                 id: Id::Extended(node_addr),
-                len: 64,
+                len: CANFD_DATA_LEN as u8,
```

Comment:
May be a typo: I thought it would be bytes.len()?
```suggestion
                len: bytes.len(),
```

Reply:
fixed

Reply:
please confirm that this change is correct - I was guessing, and wanted to defer to you on the decision on whether it would be CANFD_DATA_LEN or bytes.len().